test/schema/fixtures/ElementValue.json

```diff
@@ -6,13 +6,13 @@
     "$ref": "http://standardhealthrecord.org/schemas/shr/test/Simple.schema.json"
   },
   {
+    "$ref": "http://standardhealthrecord.org/schemas/shr/types.schema.json#/definitions/string",
```

Comment:
I won't call out every case, but here's another instance where I'd prefer to see the `$ref` listed below the description.  Please do that in _all_ cases in these fixtures.